src/editor/codeeditor.cpp

```diff
@@ -202,6 +234,216 @@ void CodeEditor::setZoom(int zoomVal)
     zoom(zoomVal - mCurrentZoom);
 }
 
+void CodeEditor::textChangedInTheOneLine()
+{
+    emit(textChangedInLine(this->textCursor().blockNumber() + 1));
+}
+
+AddCommentButton *CodeEditor::getCommentButtonByIndex(const int line)
+{
+    for (int i = 0; i < mCommentsVector.size(); i++)
```

Comment:
STL algorithms defined in the algorithm header can be used with Qt containers. You can use std::find_if and lambda expression

Reply:
Ok, I will rewrite it